core/app/models/spree/tax_rate.rb

```diff
@@ -142,8 +142,8 @@ def potentially_applicable?(order_tax_zone)
       self.zone == order_tax_zone ||
       # If the rate's zone *contains* the order's tax zone, then it's applicable.
       self.zone.contains?(order_tax_zone) ||
-      # 1) The rate's zone is the default zone, then it's always applicable.
-      (self.included_in_price? && self.zone.default_tax)
+      # 1) The rate's zone is in the default zone, then it's always applicable.
+      (self.included_in_price? && self.zone.contains?(Spree::Zone.default_tax))
```

Comment:
Yes. If your default zone is Germany, and you have a tax rate defined for Europe, and then you try to ship outside Europe, you want to be aware that there's a European tax rate in your price. 

I deliberately chose a test setup with hairy cases like this, and reverting this to the original breaks seven specs: https://gist.github.com/mamhoff/512dae674e2437133eec